harness/determined/pytorch/deepspeed/_deepspeed_context.py

```diff
@@ -0,0 +1,386 @@
+import contextlib
+import importlib.util
+import json
+import logging
+import os
+from typing import Any, Dict, Iterator, List, Optional, Set, Tuple, Type, Union, cast
+
+import torch
+import torch.nn as nn
+from deepspeed import PipelineEngine
+from deepspeed.runtime.config_utils import dict_raise_error_on_duplicate_keys
```

Comment:
Please stick to the [google style guide for python imports](https://google.github.io/styleguide/pyguide.html#22-imports); no importing symbols directly.